Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -73,10 +73,14 @@ internal extension Valet {
 
 class ValetIntegrationTests: XCTestCase
 {
-    static let identifier = Identifier(nonEmpty: "valet_testing")!
+    static let identifier = Valet.sharedAccessGroupIdentifier
     let valet = Valet.valet(with: identifier, accessibility: .whenUnlocked)
```

Comment:
Is this `valet` still necessary?

Reply:
We still need this because we have tests that are exercising the differences between configurations. However, the naming of this property did not make this obvious at all. I've updated for clarity. Good catch.